src/config/ingress-nginx/values.helm.yaml

```diff
@@ -22,6 +22,11 @@ controller:
       annotations:
         prometheus.io/scrape: "true"
         prometheus.io/port: "10254"
+  tolerations: # Schedule the ingress controller on our user node pool
```

Comment:
I wasn't sure about that, but I think I agree that it makes sense to host the ingress controller which is defacto used only to expose the workload, in the workload node pool as well.

Reply:
yeah I think it makes a lot of sense, because this way you at least have a decent number of requests that will not require an additional network hop from ingress to the workload pods. Plus, the system load that the ingress controllers generate is directly linked to the overall request rate. Thus you probably want you ingress to not impact any of your system pods - just like with your actual workload pods